apis/core/v1alpha1/flagsourceconfiguration_types.go

```diff
@@ -49,7 +49,7 @@ const (
 	defaultEvaluator                 string = "json"
 	defaultImage                     string = "ghcr.io/open-feature/flagd"
 	// renovate: datasource=github-tags depName=open-feature/flagd/flagd
-	defaultTag             string           = "v0.6.3"
+	defaultTag             string           = "v0.7.0"
```

Comment:
does flagd supports v1alpha1 at v0.7.0?
I would remove the renovate tag since we should only target supporting beta API versions